src/test/java/teammates/test/cases/logic/InstructorsLogicTest.java

```diff
@@ -382,6 +383,39 @@ private void testVerifyIsEmailOfInstructorOfCourse() throws Exception {
         AssertHelper.assertContains("Supplied parameter was null", ae.getMessage());
     }
 
+    private void testVerifyAtLeastOneInstructorIsDisplayed() throws Exception {
+
+        ______TS("success: at least one instructor is displayed to students");
+
+        String courseId = "idOfTypicalCourse1";
+        String courseIdWithNoInstructorsDisplayed = "idOfTestingInstructorsDisplayedCourse";
+        boolean isEditedInstructorDisplayed = true;
+        boolean isOriginalInstructorDisplayed = true;
```

Comment:
Why do you need these two as variables if they are never changing? Readers will find it more confusing because they have to see up. Believe me, changing all the variables into their actual boolean counterparts will make this set of test case much more readable.